cnxeasybake/oven.py

```diff
@@ -152,8 +152,10 @@ def update_css(self, css_in=None, clear_css=False):
             if rule.type == 'qualified-rule':
 
                 selectors = parse(rule.prelude, extensions=extensions)
-                decls = parse_declaration_list(rule.content,
-                                               skip_whitespace=True)
+                decls = [d for d in
+                         parse_declaration_list(rule.content,
+                                                skip_whitespace=True)
+                         if d.type == 'declaration']  # Could also be a comment
```

Comment:
I almost made it a loop, but I actually want a list, so it'd have had to initialize an empty list, then append to it, etc.